tests/test_model.py

```diff
@@ -60,7 +64,7 @@ def location_descriptors():
         {
             'id': 'NC_000001.11:15455-15566',
             'type': 'LocationDescriptor',
-            'value': {
+            'location': {
                 'sequence_id': 'ncbi:NC_000001.11',
                 'interval': {'start': 15455, 'end': 15566},
```

Comment:
Looks like you're using a SimpleInterval here. We should stick to using SequenceIntervals.